src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -151,5 +151,8 @@
     public static final String RIGHT_PARENTHESIS = ")";
     public static final String COLON = ":";
     public static final String SLASH = "/";
+    public static final String BACKSLASH = "\\";
```

Comment:
since this is only used by the unit tests should it be in the CUPdpTestConstants?

Reply:
Ok, I will move this to CUPdpTestConstants.